go.mod

```diff
@@ -264,3 +264,6 @@ require (
 )
 
 replace github.com/mholt/archiver/v3 v3.5.1 => github.com/anchore/archiver/v3 v3.5.2
+
+// this is a breaking change, so we need to pin the version until glebarez/go-sqlite is updated to not reference internal/libc
```

Comment:
Note: I had the order of events backwards, the `internal/libc` is the _new_ change...